### ISSUE TITLE
Departure from naming convention

### ISSUE THREAD

**Issue by MichaelAtOz** [Maxattax97/cornucopia#1]
I was looking around and noticed this:
https://github.com/Maxattax97/cornucopia/blob/b670a3bdbc1a7d5d5468b4ff7fe290b94a4e3fa4/util/vector.scad#L66

**Comment by Maxattax97** [comment]
Good catch, I was in the process of converting the source to fit the rest of the library and committed halfway to save my work. Guess I botched a Vim string replace :sweat_smile:, hoping the LSP I eventually get working will allow me to refactor instead.
